search/ui/src/main/res/layout/main.xml

```diff
@@ -30,7 +30,7 @@
     <EditText
         android:id="@+id/query"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
```

Comment:
doesn't this need to leave room for the ImageView with the clear button, though?

Reply:
If I understood you right, this is only a change to the height, not the width. The width is still the same (0dp).

Reply:
oh, oh, yes. 👍